### ISSUE TITLE
Random test fail on appveyor with PySide

### ISSUE THREAD

**Issue by t20100** [silx-kit/silx#1365]
`silx.gui.hdf5.test.test_hdf5.TestHdf5TreeModel.testDropExternalFile` failed once on appveyor with PySide with the following error:

```
======================================================================
FAIL: testDropExternalFile (silx.gui.hdf5.test.test_hdf5.TestHdf5TreeModel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\silx\venv_test\lib\site-packages\silx\gui\hdf5\test\test_hdf5.py", line 217, in testDropExternalFile
    self.assertIsInstance(model.nodeFromIndex(index), hdf5.Hdf5Item.Hdf5Item)
AssertionError: <silx.gui.hdf5.Hdf5LoadingItem.Hdf5LoadingItem object at 0x0000000018CCFE48> is not an instance of <class 'silx.gui.hdf5.Hdf5Item.Hdf5Item'>

[...]

FAILED (failures=1, skipped=47)
Command exited with code 1
```

See https://ci.appveyor.com/project/ESRF/silx/build/0.6.0.2916/job/e6fylipnsb396swt

To check if it happens again...

**Comment by vallsv** [comment]
It can be a timing issue.
For other widget, now i created a `hasPendingOperations`, it is a very convenient pattern to manage the synchronization. I dont remember if i already fix this widget in this way. I can check that..

**Comment by vasole** [comment]
Closed by #1366
